.gitconfig

```diff
@@ -0,0 +1,17 @@
+[user]
+  name = Yongmin Kim
+  email = yongmin86k@gmail.com
+[diff]
+[color.diff]
+  meta = yellow
+  new = green reverse
+  whitespace = red reverse
+[color]
+    ui = auto
+[alias]
+    st = status
+    ci = commit --verbose
+    co = checkout
+    di = diff -w
+    dc = diff --cached -w
+    noff = merge --no-ff
```

Comment:
This file should be stored in your $HOME dir. Please check https://github.com/redacademy/bcf-fall-2019/wiki/Git-Merge-Workflow

Reply:
oki

Reply:
Please remove this file from here and add it to your $HOME.

Reply:
Done😄